File: app/src/main/java/com/example/simonastojanovic/memeapp/domain/FetchLiveDataUseCase.java
package com.example.simonastojanovic.memeapp.domain;

import android.arch.lifecycle.MutableLiveData;

import com.example.simonastojanovic.memeapp.model.MemesItem;

import java.util.ArrayList;

public interface FetchLiveDataUseCase {

    MutableLiveData<ArrayList<MemesItem>> getLiveData();

}

File: app/src/main/java/com/example/simonastojanovic/memeapp/domain/FetchLiveDataUseCase.kt
package com.example.simonastojanovic.memeapp.domain

import android.arch.lifecycle.MutableLiveData
import com.example.simonastojanovic.memeapp.model.MemesItem
import java.util.*

interface FetchLiveDataUseCase {

    var memeLiveData: MutableLiveData<ArrayList<MemesItem>>

}
